src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/CaseWorkerRefController.java

```diff
@@ -80,7 +82,12 @@
 
             throw new InvalidRequestException("Caseworker Profiles Request is empty");
         }
-        return caseWorkerService.processCaseWorkerProfiles(caseWorkersProfileCreationRequest);
+        List<CaseWorkerProfile> caseWorkerProfileList =
+                caseWorkerService.processCaseWorkerProfiles(caseWorkersProfileCreationRequest);
+        return ResponseEntity
+                .status(201)
+                .body(caseWorkerProfileList);
```

Comment:
I think we decided not to put the caseworker details in the response. Please correct me if I am wrong

Reply:
The dependency initialisation for caseworkerServiceImpl class was getting really tough as we are not loading full application context in functional tests. However I will try for the same once more.
Even though we had decided to not alter the endpoint response, I was wondering that what difference it would make if we send back the Caseworker profiles. Anyway we are getting the CW payload in first place.

Reply:
I can understand the pain in writing the functional test for these scenarios. But I am not sure whether we can alter the response. @shreedhar-hmcts and @pksaxena01 can help us here

Reply:
As per the latest code, we are not altering the response. We are getting the IDAM id's by invoking UP with emailid as input.